### PR TITLE
USWDS-Site - POAM: June '24

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

POAM for June 2024. 

Clear dependency cache and rebuild.

## Vulnerability changes

**No changes.** 

```jsx
found 12 vulnerabilities
```

>[!NOTE]
> As of June 11th `braces` is causing a high severity vulnerability affecting 12 deps.
> There is no fix available at the time of writing this.

## Preview link

[Preview link →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-POAM-june-2024/)

## Testing and review

1.	Run ﻿npm install.
2.	Run ﻿npm run build and confirm there are no build errors.
3.	Run ﻿npm start and confirm there are no build errors.
4.	No perceived visual regressions.

## Dependency updates

No direct dependencies updated.

### CODE REVIEW

**Review by amyleadem — APPROVED** [review]
LGTM!
- [x] Run `npm install`, `npm run build`, and `npm run start` without error
- [x] Change Sass style and make sure it is reflected in the build

File: package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "postcss": "^8.4.38",
     "postcss-csso": "^6.0.1",
     "prettier": "^2.8.8",
-    "sass": "^1.77.2",
+    "sass": "^1.77.5",
```
Comment (mejiaj) [inline package.json:91]: Updated in b8928126b.

**Review by mejiaj — APPROVED** [review]
Updated SASS, no impact on `npm audit --development`.

**Comment by mejiaj** [comment]
@mahoneycm can you look into the build errors? Should be good to merge after.

**Comment by mahoneycm** [comment]
>[!IMPORTANT]
> Build errors will be resolved with link updates in #2721

**Pull request merged**